Project_outline

```diff
@@ -0,0 +1,63 @@
+# Maple-Bill Summarization and Tagging Project Document
+
+## Vy Nguyen, Gauri Bhandarwar, Andy Yang, and Weining Mai *2023-October-05*
+
+## Overview
+
+*In this document, based on the available project outline and summary of the project pitch, to the best of your abilities, you will come up with the technical plan or goals for implementing the project such that it best meets the stakeholder requirements.*
+
+### A. Provide a solution in terms of human actions to confirm if the task is within the scope of automation through AI.
+
+*To assist in outlining the steps needed to achieve our final goal, outline the AI-less process that we are trying to automate with Machine Learning. Provide as much detail as possible.*
+
+We are trying to automate the process of summarizing and tagging Massachusetts bills. For example, there are hundreds of bills here: https://malegislature.gov/Laws/GeneralLaws/PartI/TitleI/Chapter1/Section1, and our goal will be building a system that summarizes all of these bills into more digestible pieces of texts, as well as tagging each bill so that we can separate them into different categories.
```

Comment:
Incorrect reference. The URL is to MA law. Bills are what are being proposed before they get adopted into law. You will use the law as context for the bills however.